src/mpid/ch4/netmod/ofi/ofi_send.h

```diff
@@ -28,7 +28,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_send_lightweight(const void *buf,
                                         cq_data,
                                         MPIDI_OFI_av_to_phys(addr, vni_src, vni_dst),
                                         match_bits),
-                         tinjectdata, comm->hints[MPIR_COMM_HINT_EAGAIN]);
+                         vni_src, tinjectdata, comm->hints[MPIR_COMM_HINT_EAGAIN]);
```

Comment:
Here is what I mentioned above, confusing with ofi_event.c, using `vni_src` for ofi cqread.

Reply:
`fi_cqread` is responsible for both send events and recv events, I believe it is using degenerate vci/vni because of that. Only when the context gets differentiated into send_event or recv_event, we refer to the vni as vni_src or vni_dst according to the message semantics.

Reply:
`fi_cqread` only progresses local completion. So `vci_local` in `RETRY` macro is enogh.